package.json

```diff
@@ -46,7 +46,7 @@
     "setup-dev": "yarn dev-build && yarn rm-dev-link && yarn link-dev && yarn rm-aa-dev-link && yarn link-aa-dev",
     "split-e2e-tests": "yarn ts-node ./scripts/split-e2e-tests.ts && git add .circleci/config.yml",
     "test-changed": "nx affected --target=test",
-    "test-ci": "nx run-many --target=test --all -- --ci --runInBand",
+    "test-ci": "nx run-many --target=test --all -- --ci --runInBand --parallel=4",
```

Comment:
Had to ease resource pressure here as it became flaky. Other steps seem to be fine with default 8.

![Uploading image.png…]()